components/QuickLinks.tsx

```diff
@@ -0,0 +1,63 @@
+import Link from 'next/link';
+import styled, { css } from 'styled-components';
+
+export default function QuickLinks({ $isExplore, $isFavorites }) {
```

Comment:
This typing matches what your component actually needs, since some of the props appear to be optional


```suggestion
export default function QuickLinks({
  $isExplore,
  $isFavorites,
}: {
  $isExplore?: boolean;
  $isFavorites?: boolean;
}) {
```